DocSearch.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is a demonstration for how to use ThirdAI's document search engine. It assumes you are running in the ThirdAI demo docker container. Once in the container, you can get this notebook by running\n",
+    "```\n",
+    "wget https://raw.githubusercontent.com/ThirdAILabs/Demos/main/DocSearch.ipynb\n",
```

Comment:
is this not going to run automatically? Do you think we can just have a ./download_notebook.sh?

Reply:
I think this is fine for now, I want to get something out without another pr on the container side